### PR TITLE
Add REs to Golem Grafter Archetype feats

### PR DESCRIPTION
**Pull request by DocSchlock**
Touches up the Golem Grafter Dedication feat chain and adds REs where needed.

### CODE REVIEW

File: packs/feats/golem-grafter-dedication.json [hunk 1]
```diff
@@ -33,6 +33,25 @@
...
+                "value": 1
+            },
+            {
+                "exceptions": [
+                    {
+                        "definition": [
+                            "damage:material:adamantine"
+                        ],
+                        "label": "adamantine"
+                    }
```
Comment (stwlam) [inline packs/feats/golem-grafter-dedication.json:50]: This shouldn't be a custom exception

File: packs/feats/accursed-clay-fist.json [hunk 2]
```diff
@@ -43,12 +43,18 @@
                 },
                 "group": "brawling",
                 "key": "Strike",
-                "label": "Stone Fist",
+                "label": "Accursed Clay Fist",
```
Comment (stwlam) [inline packs/feats/accursed-clay-fist.json:46]:
```suggestion
```

File: packs/feats/accursed-clay-fist.json [hunk 3]
```diff
@@ -43,12 +43,17 @@
...
                 "range": null,
                 "traits": [
                     "unarmed",
                     "nonlethal"
                 ]
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "flags.pf2e.golemGrafterDedicationCount",
```
Comment (TikaelSol) [inline packs/feats/accursed-clay-fist.json:55]:
```suggestion
                "path": "flags.pf2e.golemGrafter.featCount",
```

File: packs/feats/golem-grafter-dedication.json [hunk 4]
```diff
@@ -33,6 +33,20 @@
...
+                "path": "flags.pf2e.golemGrafterDedicationCount",
+                "value": 1
+            },
+            {
+                "exceptions": [
+                    "adamantine"
+                ],
+                "key": "Resistance",
+                "type": "physical",
+                "value": "@actor.flags.pf2e.golemGrafterDedicationCount"
```
Comment (TikaelSol) [inline packs/feats/golem-grafter-dedication.json:49]:
```suggestion
                "path": "flags.pf2e.golemGrafter.featCount",
                "value": 1
            },
            {
                "exceptions": [
                    "adamantine"
                ],
                "key": "Resistance",
                "type": "physical",
                "value": "@actor.flags.pf2e.golemGrafter.featCount"
```

File: packs/feats/iron-lung.json [hunk 5]
```diff
@@ -28,7 +28,31 @@
             "remaster": false,
             "title": "Pathfinder #153: Life's Long Shadows"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "flags.pf2e.golemGrafterDedicationCount",
```
Comment (TikaelSol) [inline packs/feats/iron-lung.json:35]:
```suggestion
                "path": "flags.pf2e.golemGrafter.featCount",
```

File: packs/feats/iron-lung.json [hunk 6]
```diff
@@ -28,7 +28,31 @@
...
+            {
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.adventure-specific-actions.Item.Exhale Poison"
+            },
+            {
+                "adjustment": {
+                    "criticalFailure": "one-degree-better",
+                    "failure": "one-degree-better",
+                    "success": "one-degree-better"
+                },
```
Comment (TikaelSol) [inline packs/feats/iron-lung.json:47]:
```suggestion
                "adjustment": {
                    "all": "one-degree-better"
                },
```

File: packs/feats/legs-of-stone.json [hunk 7]
```diff
@@ -48,6 +48,12 @@
                 "selector": "reflex",
                 "type": "status",
                 "value": 2
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "flags.pf2e.golemGrafterDedicationCount",
```
Comment (TikaelSol) [inline packs/feats/legs-of-stone.json:55]:
```suggestion
                "path": "flags.pf2e.golemGrafter.featCount",
```

File: packs/feats/quicken-heartbeat.json [hunk 8]
```diff
@@ -33,7 +32,14 @@
             "remaster": false,
             "title": "Pathfinder #153: Life's Long Shadows"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "flags.pf2e.golemGrafterDedicationCount",
```
Comment (TikaelSol) [inline packs/feats/quicken-heartbeat.json:39]:
```suggestion
                "path": "flags.pf2e.golemGrafter.featCount",
```

**Pull request merged**